src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -111,7 +115,7 @@ private abstract class ScalaCheckRunner extends Runner {
         for ((name, prop) <- props)
           executeInternal(prop, name, handler, loggers, propertyFilter)
       }
-      Array.empty[Task]
+      Array.empty[Task | Null]
```

Comment:
Overridden java method returns an array. Java arrays have type `Array[T | Null]`. Returned array must thereby be converted to have type `Array[T | Null]`.